app/helpers/application_helper/button/vm_management_console.rb

```diff
@@ -0,0 +1,7 @@
+class ApplicationHelper::Button::VmManagementConsole < ApplicationHelper::Button::Basic
+  needs :@record
+
+  def visible?
+    @record.vendor == 'ibm_power_hmc'
```

Comment:
@DavidResende0 This doesn't look right.  We should be checking for appropriate supports feature rather than a specific vendor.
What if for example VMware adds `console_url` support in the future?

Reply:
I'll probably make a followup pr then

Reply:
Yes, we need to fix this